exchange/exchange.go

```diff
@@ -402,6 +410,40 @@ func (brw *bidResponseWrapper) validateBids() (err []error) {
 	return err
 }
 
+// validateCurrency will run currency validation checks and return true if it passes, false otherwise.
+func validateCurrency(requestAllowedCurrencies []string, bidCurrency string, defaultCurrency string) (bool, error) {
+	// Make sure bid currency is a valid ISO currency code
+	if bidCurrency == "" {
+		// If bid currency is not set, then consider it's default currency.
+		bidCurrency = defaultCurrency
+	}
+	currency, cerr := currency.ParseISO(bidCurrency)
```

Comment:
This might be a little fragile. Here we are redefining `currency` from being a reference to the `currency` module to be a `Unit` from that module. I am a little surprised that go did not complain about `currency` being the wrong type here actually. In any case, it may lead to confusion having multiple definitions of `currency` in the same function. I would rename the variable to `cur` or `currencyUnit`.

Reply:
Good point, thanks !